lib/bpk-component-snackbar/README.md

```diff
@@ -0,0 +1,56 @@
+# bpk-component-snackbar
+
+> Backpack React Native snackbar component.
+
+## Installation
+
+Check the main [Readme](https://github.com/skyscanner/backpack-react-native#usage) for a complete installation guide.
+
+## Usage
+
+```js
+import React from 'react';
+import { Alert } from 'react-native';
+import Buttom from 'backpack-react-native/bpk-component-buttom';
+import { icons } from 'backpack-react-native/bpk-component-icon';
+import BpkSnackbar, { LENGTH_SHORT } from 'backpack-react-native/bpk-component-snackbar';
+
+export default () => (
+  <Buttom
+    title="Show"
+    onPress={() => {
+      BpkSnackbar.show({
+        text: 'Snackbar text',
+        duration: LENGTH_SHORT,
+        title: 'Title',
+        icon: icons.flight,
+        action: {
+          text: 'Action',
+          onPress: () => {
+            Alert.alert('Action pressed');
```

Comment:
We should maybe use `BpkAlert` here to set a good example. API is the same as it's just a wrapper around RN's `Alert` component.